src/classes/UTIL_Namespace.cls

```diff
@@ -203,6 +203,7 @@ public class UTIL_Namespace {
     * @param name : the class name to have the namespace removed or replaced.
     * @return name : the class name with namespace prefix removed or replaced.
     ********************************************************************************************************/
+    @AuraEnabled
```

Comment:
Possibly - is there a downside to adding the decorator here?  I'm a little hesitant to add pass-through methods that only reference another method without any additional logic, but open to it if it is a pattern that we see benefit from following.

Reply:
I think we have avoided it in the past, but it's worth having an answer to that question rather than just doing it reflexively. Maybe I will open up a conversation in one of the dev chats and see if there's a consensus in the broader team

Reply:
Just update, currently it seems like there's not any additional reservation about applying the decorator in the util class, so let's assume this is a-ok for the time being!

Reply:
Thanks for starting the thread to discuss! - I moved it to its own class so that we can wrap it and throw AuraHandledException if needed.